README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-B
-Base repository for COMP110 worksheet B
+
+Attempt Number = 4
+
+x = Correct Letters  
```

Comment:
The worksheet asked you to provide pseudocode for an algorithm to compute the number of correct letters, however you have produced pseudocode for the overall minigame instead. Be sure to read the worksheet carefully!